### DISCUSSION TITLE
Open a synced google directory

### DISCUSSION THREAD

**Discussion by mundorfd** [ttscoff/bunch-docs#142] [Q&A]
I haven't been able how to open up a synced directory with Google Drive. 

I have been trying something like this:

```
%Finder
- ~/Volumes/GoogleDrive/My Drive
```

I was wondering if the space in the name was an issue, but `- ~/Desktop/Screen Shots` works. Any suggestions? Thanks in advance. I discovered bunch last night listening to Mac Power Users podcast and I'm so excited about using it to set up various work environments.

**Comment by ttscoff** [comment]
Volumes isn't in your home directory, it's in the root, so you don't 
want to use the tilde. Try with just:

	%Finder
	- /Volumes/GoogleDrive/My Drive

-Brett

**Comment by mundorfd** [comment]
Thank you Brett!  And thanks so much for this tool.
